### PR TITLE
Init chrome extension

### PR DESCRIPTION
**Pull request by tianfeng92**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,25 @@
 # replay-chrome-extension
-Sauce chrome extension for puppeteer replay
+Sauce chrome extension for puppeteer replay.
+
+You can run replay test on Sauce Labs using this chrome extension. Currently it's only available in Chrome Canary.
```
Comment (alexplischke) [inline README.md:4]: This is still a POC. The repo is public, however. It'd be good to indicate the state of this plugin, so that people use it with caution. There certainly is no official support at this stage.

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+
+  handleCredential(event) {
+    event.preventDefault();
+    this.setState({token: true});
+    let region;
+    if (event.target.usRegion.checked) {
+      region = event.target.usRegion.value;
+    } else {
+      region = event.target.euRegion.value;
+    }
```
Comment (alexplischke) [inline src/App.jsx]: I know this is just a POC, but that's a code smell right there that we shouldn't built on top of (and it doesn't take more work to fix). `App.jsx` should not have to have this knowledge of some deeply nested element within another component. Defeats the purpose of having components, since you are not separating concerns. https://beta.reactjs.org/learn/managing-state provides some good insight (the entire docs are worth a read).

TLDR; you can easily pass along the information you need here from the other form. For example:
```html
<form onSubmit={() => handleCredential(this.state)}>
```

And that gives you a fairly clean:
```js
  handleCredential({username, accessKey, region}) {
    this.setState({token: true});
    // eslint-disable-next-line no-undef
    chrome.storage.session.set({
      username: username,
      accessKey: accessKey,
      region: region,
      token: true,
    });
  }
```

File: src/App.jsx [hunk 3]
```diff
@@ -0,0 +1,297 @@
...
+    // eslint-disable-next-line no-undef
+    chrome.storage.session.set({
+      username: event.target.username.value,
+      accessKey: event.target.accessKey.value,
+      region,
+      token: true,
+    });
+  }
+
+  handleConfig(event) {
```
Comment (alexplischke) [inline src/App.jsx]: See my comment above.

File: src/ConfigForm.jsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+  render() {
+    const {
+      handleConfig
+    } = this.props;
+
+      return (
+        <div className="form" id="form">
+          <form onSubmit={handleConfig}>
+            <label className="form-label">
```
Comment (alexplischke) [inline src/ConfigForm.jsx]: Seems a bit unorthodox to put a variety of child elements that aren't related to to the label text into a label element 😅

Comment (tianfeng92) [reply src/ConfigForm.jsx]: fixed

File: src/App.jsx (hunk 2 above)
Comment (tianfeng92) [reply src/App.jsx]: Thanks! Updated

File: README.md (hunk 1 above)
Comment (tianfeng92) [reply README.md:4]: Add more description in README

File: src/Credential.jsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import "./Credential.css";
+
+class Credential extends React.Component {
```
Comment (alexplischke) [inline src/Credential.jsx]: Maybe `LoginForm` or some such might be more apt?

Comment (tianfeng92) [reply src/Credential.jsx]: Sounds nicer! Let me update them
